NDTensors/src/lib/Sectors/src/abstractcategory.jl

```diff
@@ -1,45 +1,143 @@
+# This file defines the abstract type AbstractCategory
+# all fusion categories (Z{2}, SU2, Ising...) are subtypes of AbstractCategory
+
 abstract type AbstractCategory end
 
-label(c::AbstractCategory) = error("method `label` not defined for type $(typeof(c))")
+# ============  Base interface  =================
+function Base.isless(c1::C, c2::C) where {C<:AbstractCategory}
+  return isless(category_label(c1), category_label(c2))
+end
 
-function dimension(c::AbstractCategory)
-  return error("method `dimension` not defined for type $(typeof(c))")
+# =================  Misc  ======================
+trivial(x) = trivial(typeof(x))
+function trivial(axis_type::Type{<:AbstractUnitRange})
+  return GradedAxes.gradedrange([trivial(eltype(axis_type))])  # always returns nondual
+end
+function trivial(la_type::Type{<:LabelledNumbers.LabelledInteger})
+  return la_type(1, trivial(LabelledNumbers.label_type(la_type)))
+end
+function trivial(type::Type)
+  return error("`trivial` not defined for type $(type).")
 end
 
-function label_fusion_rule(category_type::Type{<:AbstractCategory}, l1, l2)
-  return error("`label_fusion_rule` not defined for type $(category_type).")
+istrivial(c::AbstractCategory) = (c == trivial(typeof(c)))
```

Comment:
```suggestion
istrivial(c::AbstractCategory) = (c == trivial(c))
```
since `trivial(x)` is defined as `trivial(typeof(x))` anyway.